README.md

```diff
@@ -318,6 +318,13 @@ override fun onNewIntent(intent: Intent?) {
 app's launch intent for a tapped notification. Adjust this example to your use-case, ensuring that 
 `Klaviyo.handlePush(intent)` is called whenever your app is opened from a notification.
 
+#### Custom Data
+Klaviyo messages can also include custom key-value pairs (custom data) for both standard and silent push notifications.
```

Comment:
nit: I feel like having the word "custom" here is a little redundant given what follows it in brackets just after